src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,339 @@
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
+    display: none;
+  }
+}
+
+.pf-c-sequence__control-m-start {
+  left: var(--pf-c-sequence__control-m-start--PositionLeft);
+  order: 1;
+  border-right: var(--pf-c-sequence__control-m-start--BorderRightWidth) solid var(--pf-c-sequence__control-m-start--BorderRightColor);
+  border-left: 0;
+}
+
+.pf-c-sequence__control-m-end {
+  right: var(--pf-c-sequence__control-m-end--PositionRight);
+  order: 3;
+  border-right: 0;
+  border-left: var(--pf-c-sequence__control-m-end--BorderLeftWidth) solid var(--pf-c-sequence__control-m-end--BorderLeftColor);
+}
+
+.pf-c-sequence__control-icon {
+  min-width: var(--pf-c-sequence__control-icon--MinWidth);
+  color: var(--pf-c-sequence__control-icon--Color);
+}
+
+// Horizontal list
+.pf-c-sequence__list {
+  display: inline-flex;
+  order: 2;
+  height: var(--pf-c-sequence__list--Height);
+  padding-bottom: var(--pf-c-sequence__list--PaddingBottom);
+  overflow-x: scroll;
+  counter-reset: sequence-number;
+}
+
+// Change default styles for the list and list items
+.pf-c-sequence__list .pf-c-sequence__item {
+  list-style-type: none;
+}
+
+// Items inside of list
+.pf-c-sequence__item {
+  max-width: var(--pf-c-sequence__item--MaxWidth);
+
+  &:not(:last-child) {
+    margin-right: var(--pf-c-sequence__item--MarginRight);
+  }
+}
+
+// Content
+.pf-c-sequence__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: var(--pf-c-sequence__content--MinHeight);
+}
+
+// Content Title
+.pf-c-sequence__content-title {
+  max-width: var(--pf-c-sequence__content-title--MaxWidth);
+  overflow: hidden;
+  font-size: var(--pf-c-sequence__content-title--FontSize);
+  font-weight: var(--pf-c-sequence__content-title--FontWeight);
+  color: var(--pf-c-sequence__content-title--Color);
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+// Content Status
+.pf-c-sequence__content-status {
+  max-width: var(--pf-c-sequence__content-status--MaxWidth);
+  overflow: hidden;
+  font-size: var(--pf-c-sequence__content-status--FontSize);
+  font-weight: var(--pf-c-sequence__content-status--FontWeight);
+  color: var(--pf-c-sequence__content-status--Color);
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+// Link
+.pf-c-sequence__link {
+  display: flex;
+  align-items: center;
+  padding:
+    var(--pf-c-sequence__link--PaddingTop)
+    var(--pf-c-sequence__link--PaddingRight)
+    var(--pf-c-sequence__link--PaddingBottom)
+    var(--pf-c-sequence__link--PaddingLeft);
+  border-bottom: var(--pf-c-sequence__link--BorderBottomWidth) solid var(--pf-c-sequence__link--BorderBottomColor);
+
+  &::before {
+    position: relative;
+    width: var(--pf-c-sequence__link--Width);
+    height: var(--pf-c-sequence__link--Height);
+    margin-right: var(--pf-c-sequence__link--MarginRight);
```

Comment:
Do you see the odd overlap that this margin creates when you inspect it? I don't know if it's hurting anything 🤔

Reply:
I do see it when the number is on hover, active, focus or current. Margin-right didnt need to be changed when the number gets bigger, because with transform it doesnt affect the other positioning. I could change the spacer size, but I wonder if that creates more room for error when the text size changes and the text might jitter. Thoughts? @matthewcarleton